libyul/ObjectParser.cpp

```diff
@@ -50,7 +50,7 @@ std::shared_ptr<Object> ObjectParser::parse(std::shared_ptr<Scanner> const& _sca
 		{
 			// Special case: Code-only form.
 			object = std::make_shared<Object>();
-			object->name = "object"_yulstring;
```

Comment:
What about the other uses of `_yulstring`? Shouldn't they be changed as well? For instance: https://github.com/ethereum/solidity/blob/678e352f2093184b5cd1faee54783cfab74978ed/libyul/backends/evm/EVMObjectCompiler.cpp#L95

Reply:
the other uses can't (yet) be replaced as they are used as identifiers of nodes in the ast